README.md

```diff
@@ -39,6 +39,11 @@ If you'd like to install a specific version, set the version here (e.g. `11.4.0-
 
     gitlab_config_template: "gitlab.rb.j2"
 
+Whether to enable Git LFS and the storage path.
```

Comment:
For these notes, the description goes after the variables (so this whole block would need to go after the next paragraph, and have the variables and description flipped).